mmv1/products/firestore/BackupSchedule.yaml

```diff
@@ -0,0 +1,114 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'BackupSchedule'
+base_url: projects/{{project}}/databases/{{database}}/backupSchedules
+self_link: 'projects/{{project}}/databases/{{database}}/backupSchedules/{{name}}'
+update_verb: :PATCH
+update_mask: true
+description: |
+  A backup schedule for a Cloud Firestore Database.
+  This resource is owned by the database it is backing up, and is deleted along with the database.
+  The actual backups are not though.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/firestore/docs/backups'
+  api: 'https://cloud.google.com/firestore/docs/reference/rest/v1/projects.databases.backupSchedules'
+skip_sweeper: true
+id_format: 'projects/{{project}}/databases/{{database}}/backupSchedules/{{name}}'
+import_format:
+  - 'projects/{{project}}/databases/{{database}}/backupSchedules/{{name}}'
+  - '{{project}}/{{database}}/{{name}}'
+  - '{{database}}/{{name}}'
+docs: !ruby/object:Provider::Terraform::Docs
+  warning: |
+    This resource creates a Firestore Backup Schedule on a project that already has
+    a Firestore database. If you haven't already created it, you may
+    create a `google_firestore_database` resource with `type` set to
+    `"FIRESTORE_NATIVE"` and `location_id` set to your chosen location.
+    If you wish to use App Engine, you may instead create a
+    `google_app_engine_application` resource with `database_type` set to
+    `"CLOUD_FIRESTORE"`. Your Firestore location will be the same as
```

Comment:
Can we remove the references to FIRESTORE_NATIVE and CLOUD_FIRESTORE here? Backup schedules work with Firestore in Datastore Mode as well.

Reply:
@rwhogg updated it :)